ThemingCodelab/.gitignore

```diff
@@ -1,10 +1,12 @@
 *.iml
 .gradle
-/local
+/local.properties
 /.idea/*
-!/.idea/runConfigurations/App.xml
```

Comment:
ignore the entire .idea file

Reply:
i still see the .idea folder in

Reply:
It's not tracked anymore and it's just deleted files in pushed code.

Reply:
the `.idea/runConfigurations` folder is still there